File: PrintDemoUsb/app/src/main/java/com/demo/printdemousb/DataBean.java
package com.demo.printdemousb;


import java.text.SimpleDateFormat;

public class DataBean {
    public int m_iSeqID = 0;
    public int m_iFunID = -1;
    private String m_strValue1 = "";
    private String m_strDataTime="";

    public DataBean(int iFunID, String strValue1){
        m_iSeqID++;
        m_iFunID=iFunID;
        m_strValue1 = strValue1;
        SimpleDateFormat sDateFormat = new SimpleDateFormat("hh:mm:ss");
        m_strDataTime = sDateFormat.format(new java.util.Date());
    }
}

File: app/src/main/java/com/example/carscandemo/DataBean.kt
package com.example.carscandemo

import java.text.SimpleDateFormat
import java.util.Date

class DataBean(iFunID: Int, strValue1: String) {
    var m_iSeqID: Int = 0
    var m_iFunID: Int = -1
    private var m_strValue1 = ""
    private var m_strDataTime = ""

    init {
        m_iSeqID++
        m_iFunID = iFunID
        m_strValue1 = strValue1
        val sDateFormat = SimpleDateFormat("hh:mm:ss")
        m_strDataTime = sDateFormat.format(Date())
    }
}